internal/pkg/aws/s3/s3.go

```diff
@@ -206,6 +208,92 @@ func (s *S3) isBucketExists(bucket string) (bool, error) {
 	return true, nil
 }
 
+// GetBucketTree retrieves the objects in an S3 bucket and creates an ASCII tree representing their folder structure.
+func (s *S3) GetBucketTree(bucket string) (string, error) {
+	listResp := &s3.ListObjectsV2Output{}
+	var err error
+
+	// isBucket check to make sure the bucket exists before proceeding.
+	exists, err := s.isBucket(bucket)
+	if err != nil {
+		return "", fmt.Errorf("unable to determine the existence of bucket %s: %w", bucket, err)
+	}
+	if !exists {
+		return "", nil
+	}
+
+	for {
+		listParams := &s3.ListObjectsV2Input{
+			Bucket:            aws.String(bucket),
+			Delimiter:         aws.String("/"),
```

Comment:
can we make delimiter "/" a constant so that we can use it all the occurrences in this file